src/sub-pages/foia-records/index.html

```diff
@@ -18,7 +18,7 @@
     {% endif %}
 
     {% if sub_page.content %}
-    <section class="initiative_description block block__flush-top">
+    <section class="sub-page_content block block__flush-top">
```

Comment:
It was added to the sub-pages template in #792, but I could see it being used add consistent styles to that first paragraph in all the sub-pages. I think they're all supposed to be at an h3 level.